templates/web/peterborough/waste/_more_services_sidebar.html

```diff
@@ -38,6 +38,11 @@ <h3>More services</h3>
     <li>
         <a href="[% c.uri_for_action('waste/bulky', [ property.id ]) %]">Book bulky goods collection</a>
     </li>
+    [% IF c.cobrand.moniker == 'peterborough' %]
```

Comment:
This is in the `peterborough` directory so doesn't need the cobrand check :)

Reply:
🤦 Don't know how I didn't notice that.

Reply:
The ticket didn't actually state where the link ought to go so I assume it is OK where it is.

Reply:
Yeah I think that's the perfect place to put it.